src/meta/src/hummock/manager/mod.rs

```diff
@@ -1504,7 +1513,10 @@ where
         assert!(prev_snapshot.current_epoch < epoch);
 
         trigger_version_stat(&self.metrics, &versioning.current_version);
-        for compaction_group_id in &modified_compaction_groups {
+        for compaction_group_id in modified_compaction_groups
+            .iter()
+            .chain(deleted_compaction_groups.iter())
+        {
             trigger_sst_stat(
```

Comment:
Because metric is no longer valid, shall we consider to use `remove_label_values` in trigger_sst_stat, which has the same signature as current `with_label_values`.

Reply:
> Because metric is no longer valid, shall we consider to use `remove_label_values` in trigger_sst_stat, which has the same signature as current `with_label_values`.

fixed